tests/account/__snapshots__/test_api.ambr

```diff
@@ -1,10 +1,10 @@
 # name: TestCreateAPIKey.test[uvloop-False-False]
   <class 'dict'> {
+    'administrator': False,
     'created_at': '2015-10-06T20:00:00Z',
     'groups': <class 'list'> [
     ],
     'id': 'foobar_0',
-    'key': 'raw_key',
```

Comment:
The `key` is returned in the create request, but no other. In the UI it is displayed just this once.

I think the best solution is to create a response model that inherits from core `APIKey` and  adds the `key` field. This will only be returned on the create endpoint.